tests/cephfs/cephfs_mirroring/validate_file_dir_stats_modification_on_mirrored_cluster.py

```diff
@@ -0,0 +1,419 @@
+        cephfs_mirror_node = ceph_cluster_dict.get("ceph1").get_ceph_objects(
+            "cephfs-mirror"
+        )
+        log.info("checking Pre-requisites")
+        if not source_clients or not target_clients:
+            log.info(
+                "This test requires a minimum of 1 client node on both ceph1 and ceph2."
+            )
+            return 1
+        fs_util_ceph1.prepare_clients(source_clients, build)
+        fs_util_ceph2.prepare_clients(target_clients, build)
+        fs_util_ceph1.auth_list(source_clients)
+        fs_util_ceph2.auth_list(target_clients)
+        source_fs = "cephfs"
+        target_fs = "cephfs"
+        target_user = "mirror_remote"
+        target_site_name = "remote_site"
+        log.info("Enable mirroring mgr module on source and destination")
+        enable_mirroring_on_source = fs_mirroring_utils.enable_mirroring_module(
+            source_clients[0]
+        )
+        if enable_mirroring_on_source:
+            log.error("Mirroring module not enabled on Source Cluster.")
+            raise CommandFailed("Enable mirroring mgr module failed")
+        enable_mirroring_on_target = fs_mirroring_utils.enable_mirroring_module(
+            target_clients[0]
+        )
+        if enable_mirroring_on_target:
+            log.error("Mirroring module not enabled on Target Cluster.")
+            raise CommandFailed("Enable mirroring mgr module failed")
+        log.info("Create a user on target cluster for peer connection")
+        target_user_for_peer = fs_mirroring_utils.create_authorize_user(
+            target_fs,
+            target_user,
+            target_clients[0],
+        )
+        if target_user_for_peer:
+            log.error("User Creation Failed with the expected caps")
+            raise CommandFailed("User Creation Failed")
+        log.info(f"Enable cephfs mirroring module on the {source_fs}")
+        fs_mirroring_utils.enable_snapshot_mirroring(source_fs, source_clients[0])
+        log.info("Create the peer bootstrap")
+        bootstrap_token = fs_mirroring_utils.create_peer_bootstrap(
+            target_fs, target_user, target_site_name, target_clients[0]
+        )
+        log.info(f"Token generated after peer bootstrap : {bootstrap_token}")
+        log.info("Import the bootstrap on source")
+        fs_mirroring_utils.import_peer_bootstrap(
+            source_fs, bootstrap_token, source_clients[0]
+        )
+        log.info("Get Peer Connection Information")
+        validate_peer_connection = fs_mirroring_utils.validate_peer_connection(
+            source_clients[0], source_fs, target_site_name, target_user, target_fs
+        )
+        if validate_peer_connection:
+            log.error("Peer Connection not established")
+            raise CommandFailed("Peer Connection failed to establish")
+        log.info("Create Subvolumes for adding Data")
+        log.info("Scenario 1 : ")
```

Comment:
Can we remove Scenario 1 from here as it starts later at line 223

Reply:
sure. I can remove them